go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/nginxinc/kubernetes-ingress
 
-go 1.21.3
+go 1.22.0
```

Comment:
Go version will be bumped after https://github.com/nginxinc/kubernetes-ingress/pull/5208 is merged, so this can be ignored for now.